spec/models/user_spec.rb

```diff
@@ -238,4 +238,13 @@
     email_with_name = user.email_with_name
     expect(email_with_name).to eq("'Hans Mueller' <hans@example.org>")
   end
+
+  it 'can keep general contacting information' do
```

Comment:
This basically tests that rails can store attributes. Couldn't you just test, whether it is valid when the contact information is empty?

Reply:
It failed without the migration. I think this is a valid model test. But yes, I could also test if the user is valid.